packages/meta-harness/lib/meta_harness_common.py

```
@@ -0,0 +1,943 @@
+    さらに `.local.yaml` 上書きを config-loading ルールどおり適用する）。
+    使えない場合は DEFAULTS のみを返す。
+    """
+    try:
+        orchestra_dir = os.environ.get("AI_ORCHESTRA_DIR", "")
+        core_hooks = os.path.join(orchestra_dir, "packages", "core", "hooks")
+        if os.path.isabs(core_hooks) and os.path.isdir(core_hooks) and core_hooks not in sys.path:
+            sys.path.insert(0, core_hooks)
+        from hook_common import load_package_config
+
+        loaded = load_package_config(PACKAGE_NAME, CONFIG_FILENAME, str(project_dir))
+    except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)